dbt/models/ccbor/docs.md

```diff
@@ -0,0 +1,9 @@
+# appeals
+
+{% docs table_appeals %}
```

Comment:
**thought:** Realizing that maybe we made a mistake by not prefixing these table names in the docs with their respective database, since some stuff like `appeals` could definitely refer to multiple tables in the future. Fine for now, but something to fix in a later PR. Can you make an issue @wrridgeway?

Reply:
https://github.com/ccao-data/data-architecture/issues/559